src/pydata_sphinx_theme/assets/styles/sections/_sidebar-secondary.scss

```diff
@@ -16,7 +17,7 @@
   }
 
   padding: 2rem 1rem 1rem 1rem;
-  @include make-col(2);
+  width: 17rem;
```

Comment:
we should avoid hard-coding widths and things like that, so that we can standardize things more easily. What's the rationale for removing `make-col`? Are there any other effects? If I recall the `col` CSS does more than just set the width right?

Reply:
the make-col(2) class is setting up the width and max-width to be 16.6667%. And also making it explicit the flex-grow and flex-shrink should be zero.  
Since the default value of flex-grow and flex-shrink is 0 anyways, so it does not have any extra effect. And 16.6667% id making the secondary sidebar quite narrow with the HTML changes done in this PR. 17rem was looking okay, which is also used by sphinx-basic-ng. 
Good point, I will make it a variable.